cmd/meroxa/root/apps/create.go

```diff
@@ -61,15 +65,31 @@ func (c *Create) Flags() []builder.Flag {
 }
 
 func (c *Create) ParseArgs(args []string) error {
-	if len(args) > 0 {
-		c.args.Name = args[0]
+	if len(args) < 1 {
+		return errors.New("requires an application name")
 	}
+	c.args.Name = args[0]
 	return nil
 }
 
+func (c *Create) getLang() (string, error) {
+	if path := c.flags.Path; path != "" {
```

Comment:
what if path is pwd... we still need that to be specified as a flag?

Reply:
> what if path is pwd... we still need that to be specified as a flag?

@janelletavares That's a great question. I'm making specifying the path mandatory (notice I'm not reading from pwd) but maybe we could make that change.

> what if the supplied language is different from the app.json language?

Language specified in the app.json currently takes precedence (i.e. if both provided we ignore the currently ignore `--lang`